pkg/collector/collectors.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
```

Comment:
As far as I understand the go guidelines, `collectors.go` would need to be `collector.go` as well, right? I am sorry for the nit picking.

Reply:
Oh no, not nitpicking at all I forgot to rename this!